src/pybind/mgr/prometheus/module.py

```diff
@@ -1440,23 +1455,29 @@ def serve(self) -> None:
         cherrypy.config.update({
             'server.socket_host': server_addr,
             'server.socket_port': server_port,
-            'engine.autoreload.on': False
+            'engine.autoreload.on': False,
+            'request.show_tracebacks': False
         })
 
         module = self
 
         class Root(object):
             @cherrypy.expose
             def index(self) -> str:
-                active_uri = module.get_active_uri()
-                return '''<!DOCTYPE html>
+                standby_behaviour = module.get_module_option('standby_behaviour')
```

Comment:
This line introduces a regression, as `None` is returned if nothing is set for `standby_behaviour`. Proposed fix in https://github.com/ceph/ceph/pull/43904.

Reply:
As far as I understood (because I had the default value in there in the beginning, see https://github.com/ceph/ceph/pull/43464#discussion_r731188564) this should have been handled by the default value set in the option definition?